File: 11th grade: Machine Learning/Miscellaneous/Julia/sum/sum.jl


function sum(n)
    total = 0
    for i in 1:n
        total += i
    end
    return total
end

start = time_ns()

for i in 1:10
    sum(10^6)
end

end_time = time_ns()

print("Julia : ", (end_time - start) / 10^10)

File: 11th grade: Machine Learning/Miscellaneous/Julia/sum/sum.py
import time

def sum(n):
    total = 0
    for i in range(1, n+1):
        total += i
    return total



start = time.time()

for _ in range(10):
    sum(10 ** 6)

end_time = time.time()

print("Python:", (end_time - start) / 10)